i18n/en.json

```diff
@@ -1810,7 +1810,7 @@
   "emoji_picker.people": "People",
   "emoji_picker.places": "Places",
   "emoji_picker.recent": "Recently Used",
-  "emoji_picker.search": "Search",
+  "emoji_picker.search": "Search Emoji",
```

Comment:
We should update this where it's used in `components/emoji_picker/emoji_picker.jsx`